src/router/metadata/metadataHandler.test.ts

```diff
@@ -431,6 +446,45 @@ test('R4: FHIR Config V4 without search', async () => {
         updateCreate: configHandler.config.profile.genericResource!.persistence.updateCreateSupported,
     };
     expect(response.resource.rest[0].resource[0]).toMatchObject(expectedResourceSubset);
+    expect(response.resource.rest[0].resource[0]).toMatchInlineSnapshot(`

```

Comment:
What's the difference between `toMatchInlineSnapshot` and `toMatchObject` here except the formatting different? Looking at the Jest document, it only talks about using `toMatchInlineSnapshot` for front end testing.

Reply:
I updated to remove the `toMatchObject` expectation since it is redundant now.

`toMatchInlineSnapshot` matches the input against the string snapshot that appears below. It's mostly used in frontend due to it's ties with jest + react, but it can be used for anything that can be serialized into a string (i.e. JSON)

I added the snapshot to test that the `fakeOperation` is present. I could have updated the `expectedResourceSubset` instead but I find snapshots to be easier to understand in this case.
i.e., for `toMatchObject(expectedResourceSubset)` you need to see the definition of `expectedResourceSubset` and then the definition of the `makeOperation` function to understand the expected result. Snapshots immediately tell you the expected JSON. 

https://jestjs.io/docs/snapshot-testing#does-snapshot-testing-only-work-with-react-components

Reply:
Emm that makes sense, good call